sql-odbc/README.md

```diff
@@ -14,6 +14,10 @@ The driver is compatible with ODBC 3.51.
   |  Windows    |  Windows 10   | 32-bit, 64-bit |
   |  MacOS    |  Catalina 10.15.4, Mojave 10.14.6 | 64-bit |
 
+## Connectors
+
+**Power BI Desktop**: [`OdfeSqlOdbcPBIConnector.mez`](./src/PowerBIConnector/bin/Release/OdfeSqlOdbcPBIConnector.mez)
+
```

Comment:
add taco connector link as well if **Connectors** section is not specific to Power BI